client/src/components/AuditAdmin/Progress/Progress.tsx

```diff
@@ -60,6 +61,19 @@ const totalFooter = <T extends object>(headerName: string) => (
   info: TableInstance<T>
 ) => sum(info.rows.map(row => row.values[headerName])).toLocaleString()
 
+// We count the number of batch-contest pairs with discrepancies, not the number
+// of batches with discrepancies.
+const countDiscrepanciesForJurisdiction = (
+  discrepancies: DiscrepanciesByJurisdiction,
+  jurisdictionId: string
+) => {
+  return sum(
+    Object.entries(discrepancies[jurisdictionId] ?? {}).map(
+      ([_, contestDiscrepancies]) => Object.keys(contestDiscrepancies).length
+    )
```

Comment:
nit: maybe `Object.values()` since the key isn't used?

```
Object.values(discrepancies[jurisdictionId] ?? {}).map(
      contestDiscrepancies => Object.keys(contestDiscrepancies).length
    )
```